content/usgs-packages/dataRetrieval_readNWIS.Rmd

```diff
@@ -327,9 +342,11 @@ head(or_site_do)
 # Parameter: mercury and lead in micrograms/liter, 71890 and 01049
 # Begin date: January 1, 1972
 
-oh_site_cwa <- readNWISqw(siteNumbers="03237280", parameterCd=c("71890", "01049"), startDate="1972-01-01")
+oh_site_cwa <- readNWISqw(siteNumbers="03237280", 
+                          parameterCd=c("71890", "01049"),
+                          startDate="1972-01-01")
 nrow(oh_site_cwa)
-head(oh_site_cwa)
+head(oh_site_cwa[,c("parm_cd","sample_dt","result_va")])
```

Comment:
same thing from `or_site_do` - can we show `ncol` to explain why we are selecting just 3